controller/gui/style.css

```diff
@@ -1,43 +1,49 @@
-body {
+/* Template */
+
+#d-Container {
     font-family: monospace;
     background-color: white;
     color: black;
     margin: 5vh 5vw 5vh 5vw;
     padding: 0;
-}
 
-dl > dt {
-    font-weight: bold;
+    display: grid;
+
+    grid-template-columns: 1fr 10fr;
+    grid-template-rows: 1fr 13fr 1fr;
+    grid-template-areas:
+        "header header"
+        "menu content"
+        "footer footer";
+    align-items: stretch;
+
+    height: 90vh;
 }
 
-#header {
+#d-Header {
     grid-area: header;
 }
 
-.menu-item {
-    display: block;
+#d-Content {
+    grid-area: content;
+    place-self: center;
+    font-size: 2em;
+    width: 90%;
+    height: 90%;
+    overflow: scroll;
     margin: 10px;
 }
 
-.menu-item-icon {
-    width: 100%;
-}
-
-.menu-item-icon:hover {
-    filter: invert(70%);
-}
 
-.menu-item.active .menu-item-icon {
-    filter: invert(70%) sepia(70%) saturate(100%);
+#d-Footer {
+    grid-area: footer;
+    text-align: right;
+    margin: 20px;
 }
 
-.hidden-link {
-    text-decoration: none;
-    cursor: text;
-    color: black;
-}
+/* Menu */
 
-#menu {
+#d-Menu {
```

Comment:
While at it: I don't see any benefit to using `id` selectors here. These could just be classes.

Reply:
I was not sure if it was BEM related or not, I am also in favor of using classes.